src/main/java/com/gmail/woosay333/onlinebookstore/service/BookService.java

```diff
@@ -1,13 +1,17 @@
 package com.gmail.woosay333.onlinebookstore.service;
 
-import com.gmail.woosay333.onlinebookstore.dto.BookResponseDto;
+import com.gmail.woosay333.onlinebookstore.dto.BookDto;
 import com.gmail.woosay333.onlinebookstore.dto.CreateBookRequestDto;
 import java.util.List;
 
 public interface BookService {
-    BookResponseDto save(CreateBookRequestDto bookRequestDto);
+    BookDto save(CreateBookRequestDto bookRequestDto);
```

Comment:
```suggestion
    BookDto create(CreateBookRequestDto bookRequestDto);
```
because `repository.save()` is used for both create and update which makes this method name confusing